tst/testspecial/debug-var.g.out

```diff
@@ -118,32 +118,28 @@ brk_2>
 brk_2> # check coding of dvars
 brk_2> function() return unbound_higher; end;
 Error, Variable: <debug-variable-1-4> cannot be used here in
-  <corrupted statement>  called from 
```

Comment:
I think this is actually a bug fix :-). For a very fringe and remote bug, but still... I've always wondered where those "corrupted statement" came from, now it's clear: from the fact that we throw an error while in the middle of coding.